README.md

```diff
@@ -1 +1,212 @@
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 JavaScript Capstone Project <a name="about-project"></a>
+
+> In this project, we are setting up a Ecommerce website with basic functionalities using JavaScript modules, Webpack and APIs.
+**My Shop** is a basic product listing page of an E-commerce website.
+The website display the list of products and allows users to like their favorite product, their can also chech the comments of others users about a specific product and finally the user can comment a product.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Building</summary>
+  <ul>
+    <li><a href="https://www.javascript.com/">JavaScript</a></li>
+    <li>HTML 5</li>
+    <li>CSS 3</li>
+    <li>ES6</li>
+  </ul>
+</details>
+<details>
+  <summary>Bundle</summary>
+  <ul>
+    <li><a href="https://webpack.js.org/">WebPack</a></li>
+    <li><a href="https://www.npmjs.com/">NPM</a></li>
+  </ul>
+  </details>
+<details>
+  <summary>Tests</summary>
+  <ul>
+    <li><a href="https://jestjs.io/">Jest</a></li>
+    <li><a href="https://babeljs.io/">Babel JS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://webpack.js.org/configuration/dev-server/">WebPack Dev Server</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://fakestoreapi.com/">Fakestore API</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **Like the favorite Product**
+- **Check for comments of other users in a pop-up**
+- **Add new comment to the favorite products**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](https://vanelnw.github.io/JavaScript-Capstone/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Github account.
+- Code editor (Visual Studio Code, Atom, etc.)
+- WebPack
+- Jestjs
+- Babeljs
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+- cd my-folder
+- git clone https://github.com/vanelnw/JavaScript-Capstone.git
+
+
+### Install
+
+Install this project with:
+
+- You'll need to install Webpack on your root directory by using the following command on your terminal:
+
+    `npm install webpack webpack-cli --save-dev`
+- Now you can use the npm run build command to create your new files on the dist folder by using the following command:
+
+    `npm run build`
+
+### Usage
+
+To run the project, execute the following command:
+
+    `npm start`
+
+### Run tests
+
+To run tests, run the following command:
+
+    `npm test`
+
+
```

Comment:
- [ ] OPTIONAL
 Kindly ensure to add the _Future Features_ to your ReadMe, which should include 1-3 potential features to be added